cmd/repo-updater/repoupdater/server.go

```diff
@@ -65,6 +66,7 @@ func (s *Server) Handler() http.Handler {
 	mux.HandleFunc("/sync-external-service", trace.WithRouteName("sync-external-service", s.handleExternalServiceSync))
 	mux.HandleFunc("/enqueue-changeset-sync", trace.WithRouteName("enqueue-changeset-sync", s.handleEnqueueChangesetSync))
 	mux.HandleFunc("/schedule-perms-sync", trace.WithRouteName("schedule-perms-sync", s.handleSchedulePermsSync))
+	mux.HandleFunc("/ext-svc-namespaces", trace.WithRouteName("ext-svc-namespaces", s.handleQueryExternalServiceNamespaces))
```

Comment:
We have another route further up, called `sync-external-service`, so I'd stay consistent with the naming:

```suggestion
	mux.HandleFunc("/external-service-namespaces", trace.WithRouteName("ext-svc-namespaces", s.handleQueryExternalServiceNamespaces))
```